spm_funcs.py

```diff
@@ -55,6 +55,12 @@ def get_spm_globals(fname):
         SPM global metric for each 3D volume in the 4D image.
     """
     # +++your code here+++
+    img = nib.load(fname)
+    data = img.get_fdata()
+    spm_vals = np.arange(data.shape[-1])
+    for vol in range(data.shape[-1]):
+        spm_vals[vol] = (spm_global(data[...,vol]))
```

Comment:
Unnecessary parentheses
```suggestion
        spm_vals[vol] = spm_global(data[...,vol])
```